### PR TITLE
feat: Google Tag Manager (GTM) App

### PR DESCRIPTION
**Pull request by StefanFlorea00**
## What does this PR do?

There is an existing app for Google Analytics 4. We created an app that works in the same way but with Google Tag Manger integration.

**Environment**: Staging(main branch) / Production

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## How should this be tested?

-  Install the app
-  Add the app to an event, fill in the Tracking ID with a GTM container ID
-  Ensure that the GTM script is visible on the booking page (in inspect elements)
-  Preview and test if GTM events are fired successfully using the tagmanager

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

## TODO:
- [ ] copy changes and screenshot review @Jaibles @PeerRich

### CODE REVIEW

File: packages/app-store/google-tag-manager/README.md
Comment (hariombalhara) [inline packages/app-store/google-tag-manager/README.md:1]: I am not sure why this file is there. We can remove it. If it came from AppStore Cli, will fix that.

File: packages/app-store/google-tag-manager/static/1.jpeg
Comment (hariombalhara) [inline packages/app-store/google-tag-manager/static/1.jpeg:1]: We need the correct image here that demoes that the App is working. cc @PeerRich

File: yarn.lock
Comment (hariombalhara) [inline yarn.lock:1]: I am not sure why this file is there. There are no dependencies update. So, we should just uncommit this file.

Comment (StefanFlorea00) [reply yarn.lock:1]: Removed

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**